lib/Connection.js

```diff
@@ -30,68 +35,95 @@ class Connection {
    * @param {string} db
    * @param {string} user
    * @param {string} pwd
-   * @param {object} [option]
+   * @param {object} [restConfig]
    */
-  constructor(db, user, pwd, option) {
-    this.conn = {};
+  constructor(db, username, password, restConfig) {
+    // setup default values
+    this.conn = {
+      database: '*LOCAL',
+      username: '',
+      password: '',
+      xslib: 'QXMLSERV',
+      returnError: true,
+      returnXml: false,
+      transport: 'idb',
+      host: 'localhost',
+      port: 80,
+      path: '/',
+      ctl: '*here',
+      ipc: '*NA',
+      outputBuffer: 500000,
```

Comment:
I am in support of just removing `outputBuffer` completely. Always use the max output size.

Reply:
This is only really used for rest transport

Max output size would 15M right? 

Would need to set that in bytes according to [docs](http://youngiprofessionals.com/wiki/index.php/XMLSERVICE/XMLSERVICEQuick#facerest)